cmd/build/helmify/static/README.md

```diff
@@ -87,12 +87,14 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | validatingWebhookFailurePolicy                | The failurePolicy for the validating webhook                                                                                                                                                                                                        | `Ignore`                                                                  |
 | validatingWebhookCheckIgnoreFailurePolicy     | The failurePolicy for the check-ignore-label validating webhook                                                                                                                                                                                     | `Fail`                                                                    |
 | validatingWebhookExemptNamespacesLabels       | Additional namespace labels that will be exempt from the validating webhook. Please note that anyone in the cluster capable to manage namespaces will be able to skip all Gatekeeper validation by setting one of these labels for their namespace. | `{}`                                                                      |
-| enableDeleteOperations                        | Enable validating webhook for delete operations                                                                                                                                                                                                     | `false`                                                                   |
+| validatingWebhookCustomRules                  | Custom rules for selecting which API resources trigger the webhook                                                                                                                                                                                  | `{}`                                                                      |
```

Comment:
might be good to mention that `validatingWebhookCustomRules` and `enableDeleteOperations` are mutually exclusive.

Reply:
It says `Does not work with validatingWebhookCustomRules` on the next line, are you wanting more specifics than that?

Reply:
It’s only mentioned for one, but not both. If a user wanting to add the new validatingWebhookCustomRules, they may not realize their existing values for enableDeleteOperations will not work as intended anymore.